src/borg/testsuite/archiver.py

```diff
@@ -1777,6 +1769,9 @@ def raise_eof(*args):
             self.cmd('init', '--encryption=repokey', self.repository_location, exit_code=1)
         assert not os.path.exists(self.repository_location)
 
+    def test_init_encryption(self):
+        self.cmd('init', self.repository_location, exit_code=2)
```

Comment:
add a comment what this is about?

or rename to test_init_requires_encryption_option to make it clearer.